modules/miraheze.py

```diff
@@ -37,14 +36,14 @@ def miraheze(bot, trigger):
 
 
 @commands('gethelp')
+@rule('[i-iI-I] need help|[c-cC-C]an someone help me|[i-iI-I] can(t|'t) login).*')
```

Comment:
This has a ‘ in the middle so closes too early. You need either changed (“ .. “) to the outside or escape the ‘

Reply:
oh I see